acquia_lift.module

```diff
@@ -2853,35 +2778,16 @@ function acquia_lift_command_messagebox($message, $seconds = 0) {
 }
 
 /**
- * Returns an AJAX command to toggle the page variation element selector.
+ * Returns an AJAX command to toggle the element variation element selector.
  *
  * @param bool $start
- *   True to start the page variation mode, FALSE to stop it.
+ *   True to start the element variation mode, FALSE to stop it.
  */
-function acquia_lift_command_page_variation_toggle($start = TRUE) {
+function acquia_lift_command_element_variation_toggle($start = TRUE) {
```

Comment:
I would make this one acquia_lift_command_variation_toggle($start = TRUE).  That way it is lined up with the JS command that it calls.  The only reason we had both before was because they both called the acquia_lift_command_variation_toggle JS command and specified the type.